src/main/java/frc/robot/Robot.java

```diff
@@ -35,6 +35,7 @@ public void robotPeriodic() {
 
   @Override
   public void disabledInit() {
+    CommandScheduler.getInstance().schedule(RobotContainer.disabledLights);
```

Comment:
You *should* just be able to call RobotContainer.disabledLights.schedule() here, shouldn't you?